toss/trajectory/compute_trajectory.py

```diff
@@ -66,9 +66,33 @@ def compute_trajectory(x: np.ndarray, args, func: Callable) -> Union[bool, list,
         # Rearrange maneuvers into increasing order of time of execution
         maneuver_times = integration_intervals[1:-1]
         correct_order_of_maneuvers = np.argsort(maneuver_times)
-        integration_intervals[1:-1] = maneuver_times[correct_order_of_maneuvers]
+        maneuver_times = maneuver_times[correct_order_of_maneuvers]
+        integration_intervals[1:-1] = maneuver_times
         dv_of_maneuvers = dv_of_maneuvers[:,correct_order_of_maneuvers]
+
+        # Adjust for simultaneous manuevers
+        for idx in range(0,len(maneuver_times)-1):    
+            if idx+1 == len(maneuver_times) or idx == len(maneuver_times):
+                break
+            duplicate_idx = idx
+            while maneuver_times[duplicate_idx] == maneuver_times[duplicate_idx+1]:
+                dv_of_maneuvers[:,idx] += dv_of_maneuvers[:,duplicate_idx+1]
+                duplicate_idx +=1
+
+                if duplicate_idx+1 == len(maneuver_times):
+                    break
+            
+            # Remove duplicate manuver time (and corresponding maneuver)
+            maneuver_times = np.delete(maneuver_times, np.arange(idx+1,duplicate_idx+1,1))
+            dv_of_maneuvers = np.delete(dv_of_maneuvers, np.arange(idx+1,duplicate_idx+1,1), axis=1)
```

Comment:
?? But if two maneuvers were done you cannot just discard!?!?!? the dV imparted by that maneuver would still be present?

The correct way to treat this would be an inequality constraint on the maneuvers times, I think?

Reply:
(also why is this even here, I thought this was about multiple spacecraft)

Reply:
On second reading I think you keep the DV, but then you violate the constraints on maneuver DV?

Reply:
Correct, It has so far not favored simultaneous maneuvers, however, it is still possible to violate the tot DV constraint. In general, I think that the code (and especially the udp) would benefit greatly in clarity and in avoiding confusing code by adding constraints. It is already tagged in the issue #37